packages/patternfly-4/react-charts/src/components/ChartUtils/chart-interactive-legend.ts

```diff
@@ -51,29 +43,15 @@ const getInteractiveLegendItems = ({ chartNames, omitIndex }: ChartInteractiveLe
   return result;
 };
 
-// Returns styles for interactive legend items
-export const getInteractiveLegendItemStyles = (hidden: boolean = false) => {
-  return !hidden ? {} : {
-    labels: {
-      fill: chart_color_black_500.value
-    },
-    symbol: {
-      fill: chart_color_black_500.value,
-      type: 'eyeSlash'
-    }
-  };
-};
-
 // Returns targeted events for legend 'data' or 'labels'
 const getInteractiveLegendTargetEvents = ({
   chartNames,
-  isHidden = (index) => false,
+  isHidden = () => false,
   legendName,
-  onLegendClick = (props) => null,
+  onLegendClick = () => null,
   target
 }: ChartInteractiveLegendExtInterface) => {
   if (chartNames === undefined || legendName === undefined) {
-    console.error('getInteractiveLegendTargetEvents:', 'requires chartNames and legendName to be specified');
```

Comment:
This is kinda important information to help setup interactive legends properly. Why remove it?

Reply:
Thanks for the context - I'll add this back in!  It was originally removed per the [no-console](https://eslint.org/docs/rules/no-console) rule

Reply:
Added console.error back in